src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -45,6 +45,12 @@ object ScalafixPlugin extends AutoPlugin {
         "Optional list of custom rules to install from Maven Central. " +
           "This setting is read from the global scope so it only needs to be defined once in the build."
       )
+    val scalafixScalaVersion: SettingKey[String] =
```

Comment:
I initially had a boolean that I could not find a good name for (scalafixScalaVersionMatchesBuild`?), with a default of `false` to keep `2.12` no matter what the project is set up with. I guess it would be simpler as ut really represent the intent, if only we could find a good name for it?

Reply:
I think it's good to use `scalafixScalaVersion` and we can default it to 2.12 for all project